tasks/main.yml

```diff
@@ -1,12 +1,19 @@
 ---
-- name: Install packages
+
+- include_vars: vars_RedHat_CentOS.yml
+  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+
+- include_vars: vars_SUSE_openSUSE.yml
+  when: 'ansible_os_family == "Suse" or ansible_os_family == "openSUSE Leap"'
+
+- name: "Install package(s) {{ iscsiadmin_package }}"
   yum:
```

Comment:
Is this really working on SUSE with 'yum' module?

Reply:
It is apparently. Nevertheless I have changed that to `package` now.